src/kmer.py

```diff
@@ -1,31 +1,54 @@
 """Computing kmers of a string."""
 
-
+kmer_list = []
```

Comment:
Using a global variable here means that the list will grow every time you call the function. The function will not just return the k'mers in its input. You probably want this to be a local variable.